File: 0933-number-of-recent-calls/0933-number-of-recent-calls.cs
public class RecentCounter {
    
    List<int> requests = new List<int>();
    
    public RecentCounter() {

    }
    
    public int Ping(int t) {
        requests.Add(t);
        
        var start = t - 3000;
        
        return requests.Where(r => r >= start && r <= t).ToList().Count;
    }
}

/**
 * Your RecentCounter object will be instantiated and called as such:
 * RecentCounter obj = new RecentCounter();
 * int param_1 = obj.Ping(t);
 */

File: 0933-number-of-recent-calls/0933-number-of-recent-calls.swift

class RecentCounter {

    var requests: [Int] = []

    init() {
        
    }
    
    func ping(_ t: Int) -> Int {
        
        requests.append(t)
        
        var range = (t-3000 ... t)
        
        return requests.filter { range.contains($0) }.count
    }
}

/**
 * Your RecentCounter object will be instantiated and called as such:
 * let obj = RecentCounter()
 * let ret_1: Int = obj.ping(t)
 */